elements/pfe-card/src/pfe-card.html

```diff
@@ -1,4 +1,5 @@
 <!-- pfe-card -->
+<div class="pfe-card--overlay"></div>
```

Comment:
@castastrophe I think this needs an if statement wrapped around it

Reply:
No, it’s correct as-is. It prints to the DOM and styles determine if it is visible or not.

Reply:
No, it’s correct as-is. It prints to the DOM and styles determine if it is visible or not.

Reply:
No, it’s correct as-is. It prints to the DOM and styles determine if it is visible or not.

Reply:
No, it’s correct as-is. It prints to the DOM and styles determine if it is visible or not.